.github/ISSUE_TEMPLATE.md

```diff
@@ -1,13 +1,16 @@
+##Software versions
 - React-Native VS Code extension version: 
 - VSCode version: 
 - OS platform and version: 
 - React Native version: 
-- (If relevant) Output of the Debug Console (View -> Toogle Debug Console): 
-- (If relevant) Output of the React-Native output channel (View -> Toogle Output -> Select React-Native in ListBox): 
-- (If relevant) Output of the Developer Tools console (Help -> Toggle Developer Tools -> Select Console tab): 
 
-Steps to Reproduce:
+##Outputs (Include if relevant)
+- Output of the Debug Console (View -> Toogle Debug Console): 
+- Output of the React-Native output channel (View -> Toogle Output -> Select React-Native in ListBox): 
+- Output of the Developer Tools console (Help -> Toggle Developer Tools -> Select Console tab): 
+
+##Steps to Reproduce:
 1. 
 2. 
 
-What do you expect that should've happened instead:
+##What do you expect that should've happened instead:
```

Comment:
I like @joshuaskelton's suggestion to use two new `Actual Behavior` and `Expected Behavior` paragraphs instead of this one.